normandy/recipes/models.py

```diff
@@ -102,6 +126,22 @@ def revision_id(self):
     def last_updated(self):
         return self.latest_revision.updated if self.latest_revision else None
 
+    @property
+    def filter_expression(self):
+        return self.latest_revision.filter_expression if self.latest_revision else None
+
+    @property
+    def channels(self):
+        return self.latest_revision.channels if self.latest_revision else None
+
+    @property
+    def countries(self):
+        return self.latest_revision.countries if self.latest_revision else None
+
+    @property
+    def locales(self):
```

Comment:
So at what point do we make a helper for this?

```python
@latest_revision_property
def countries(self, latest_revision):
    return latest_revision.countries
```

and make it return `None` without calling the function if `latest_revision` doesn't exist?